pkg/auth/rbac.go

```diff
@@ -0,0 +1,76 @@
+package auth
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Inits the RBAC Config struct
+func initRbacConfig() RBACconfig {
+	return RBACconfig{
+		allowedApp:     []string{"EnvironmentLock", "EnvironmentApplicationLock", "Deploy", "Undeploy", "EnvironmentFromApp"},
```

Comment:
Please make the name `EnvironmentFromApp` consistent and use `EnvironmentFromApplication`

Reply:
Fixed it.